enterprise/coderd/prebuilds/metricscollector.go

```diff
@@ -10,30 +10,33 @@ import (
 
 	"github.com/coder/coder/v2/coderd/database"
 	"github.com/coder/coder/v2/coderd/database/dbauthz"
+	"github.com/coder/coder/v2/coderd/prebuilds"
 )
 
 var (
 	CreatedPrebuildsDesc   = prometheus.NewDesc("coderd_prebuilds_created", "The number of prebuilds created.", []string{"template_name", "preset_name"}, nil)
```

Comment:
is there a reason why `desc` are exposed?

Reply:
nope. fixed